react/src/rebate_application_form/ApplicationFormContainer.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import React, { useEffect, useRef, useState } from 'react';
+import ApplicationFormPage from './components/ApplicationFormPage';
+import ROUTES_APPLICATION from './routes';
+
+
+const ApplicationFormContainer = () => {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [details, setDetails] = useState({});
+
+  const handleInputChange = (event) => {
+    const { value, name } = event.target;
+    setDetails({
+      ...details,
+      [name]: value,
+    });
+  };
+  const handleSubmit = () => {
+    axios.post(ROUTES_APPLICATION.SAVE, { details })
+      .then((response) => { console.log(response); });
+  };
+  useEffect(() => {
+    setLoading(true);
+    // do an axios get for whatever informaion we need to retrieve for the form to be filled
```

Comment:
Do we need this effect?

Reply:
no, i thought we might be retrieving data for the form but i'll take it out until we do